labs/src/main/scala/org/scalalabs/basic/lab03/PatternMatchingExercise.scala

```diff
@@ -55,7 +71,10 @@ object PatternMatchingExercise {
    *    otherwise return `None`
    */
   def older(p: Person): Option[String] = {
-    error("fix me")
+    p match {
+        case _ if(p.age > 30) => Some(p.name)
+        case _ => None
+    }
```

Comment:
Does the job it needs to and does it well! It might be useful to include a _ case in the first pattern match.